2020-05-01 18:57:54,601 ERROR: Exception on /auth/login [GET] [in c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\auth\routes.py", line 16, in login
    return render_template('auth/login.html',title='Sign In', form=form)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\auth\login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 3, in block "doc"
    <html{% block html_attribs %}{% endblock html_attribs %}>
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 11, in block "html"
    {%- endblock metas %}
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 13, in block "body"
    {%- block styles %}
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\base.html", line 48, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\auth\login.html", line 19, in block "app_content"
    <p>New User?<a href="{{ url_for('register') }}"> Sign Up Here!</a></p>
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\helpers.py", line 358, in url_for
    endpoint, values, method=method, force_external=external
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'register'. Did you mean 'auth.register' instead?
2020-05-01 18:58:07,469 ERROR: Exception on /auth/login [GET] [in c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\auth\routes.py", line 16, in login
    return render_template('auth/login.html',title='Sign In', form=form)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\auth\login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 3, in block "doc"
    <html{% block html_attribs %}{% endblock html_attribs %}>
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 11, in block "html"
    {%- endblock metas %}
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 13, in block "body"
    {%- block styles %}
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\base.html", line 48, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\templates\auth\login.html", line 19, in block "app_content"
    <p>New User?<a href="{{ url_for('register') }}"> Sign Up Here!</a></p>
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\flask\helpers.py", line 358, in url_for
    endpoint, values, method=method, force_external=external
  File "c:\users\andrew~1\envs\flask-kickoff\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'register'. Did you mean 'auth.register' instead?
2020-05-01 18:58:48,284 INFO: Kickoff startup [in C:\Users\Andrew Yu\Documents\Prog\Projects\kickoff\app\__init__.py:60]
